cruise.umple/src/en.error

```diff
@@ -207,7 +207,8 @@
 1014: 3, "http://manual.umple.org/?W1014ExcludedMethodNotFoundInjection.html", Excluded method '{0}' cannot be found. The exclusion was ignored. ;
 # Issue 1483
 1015: 3, "http://manual.umple.org/?BasicStateMachines.html", Methods in standalone state machines might not be correctly generated. State machines can be used in traits or put directly in a class to achieve the same effect. ;
-
+1016: 3, "http://manual.umple.org/?PageBeingDeveloped.html", Extra Open Or Closing Bracket encountered. Processing terminated. All code beyond this point is ignored. ;
+1020: 3, "http://manual.umple.org/?PageBeingDeveloped.html", Extra closing curly bracket encountered. Processing terminated. All code beyond this point is ignored. ;
```

Comment:
Please take out 1020 and change 1016 so the 'Processing terminated and following is no longer there.